src/EA.Weee.Web/Areas/Admin/Controllers/ProducerSubmissionController.cs

```diff
@@ -129,16 +134,87 @@ public async Task<ActionResult> AddPaymentDetails()
             return View();
         }
 
+        [AdminSmallProducerSubmissionContext]
         [HttpGet]
-        public async Task<ActionResult> RemoveSubmission()
+        public async Task<ActionResult> RemoveSubmission(string registrationNumber, int year)
```

Comment:
@stevelee-civica  This one probably doesnt need to be async.
I wonder if when the submission.AuthorisedRepresentitiveData  is present we use that as the name, might be worth confirming with James.

Reply:
updated